src/styles/StyleUtils.js

```diff
@@ -1302,6 +1302,22 @@ function getCheckboxContainerStyle(size, borderRadius) {
     };
 }
 
+/**
+ * Returns style object for the dropbutton height
+ * @param {Boolean} isLargeButton
+ * @returns {Object}
+ */
+function getButtonHeight(isLargeButton) {
```

Comment:
This function is specific to drop down button right? can we rename the function to be specific, current naming sounds like its a generic function